src/loaders/sdf.cpp

```diff
@@ -62,10 +62,19 @@ tue::config::NodeType getSDFNodeType(const std::string& element_name)
 
 bool loadFromSDFElement(const TiXmlElement& element, ReaderWriter& config, const NodeType node_type)
 {
+    std::string element_name = element.ValueStr();
```

Comment:
Why do you construct that here, and use it 20 lines further down? Also, as it's used one time only, wouldn't it be simpler to add a comment saying you pass the name in rather than making new `std::string` values

Reply:
I used it in multiple printing statements. But these were only used for debugging. So now it isn't needed anymore.